.gitignore

```diff
@@ -21,3 +21,11 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# ignore type definitions
+*.d.ts
```

Comment:
Please remove `d.ts` 
`.map`
 from ignore

Reply:
These produce lots of noise on git status, and should part of the release's DevOps process.
However you are in control of the release - so removed them.